### DISCUSSION TITLE
Support for Elasticsearch with an Elasticsearch driver

### DISCUSSION THREAD

**Discussion by vijamesDD** [flamingock/mongock#22] [Q&A] [answered]
Hi, 

I'd love to use mongock on my current project which uses Elasticsearch db. The [docs ](https://docs.mongock.io/v5/driver/index.html#drivers-available) say that an elasticsearch driver is "coming soon", do you have any timeline on this? I note that [this issue ](https://github.com/mongock/mongock/issues/492) was closed as well 👀 

Thanks very much!

**Comment by dieppa** [comment, marked as answer]
Hello @vijamesDD  ,

as mentioned in [this issue](https://github.com/mongock/mongock/issues/492), we dismissed the Elasticsearch's driver because **we are working on a new cloud based solution which will support any database**(just to be clear, this doesn't mean your data would be stored in the cloud, just the Mongock's data).

We don't have a definitive release date yet, but it will be soon.  If you are interested in being part of the beta release program, please send us an email to development@mongock.io.

Note that you don't need a ElasticSearch driver, if you have access to a MongoDB(and compatible databases), DynamoDB or CouchDB. You can configure Mongock to use one of these databases to store its metadata and then use your change units to handlethe ElasticSearch changes.

**Alternatively we are more than happy in assisting you in case you want to raise a PR providing a new driver for ElasticSearch.**  

Thanks

**Reply by vijamesDD** [reply]
Thanks @dieppa , appreciate the response!
